lib/vmdb/deprecation.rb

```diff
@@ -1,7 +1,7 @@
 module Vmdb
   class Deprecation
     def self.instance
-      @instance ||= ActiveSupport::Deprecation.new("D-release", "ManageIQ").tap { |d| d.behavior = default_behavior }
+      @instance ||= ActiveSupport::Deprecation.new("F-release", "ManageIQ").tap { |d| d.behavior = default_behavior }
```

Comment:
(Coming in from #10066) Ahh I see. Yes, these removals should be backported to Darga with this changed to "Euwe release" and `master` should be updated to "F-release"